integration-tests/src/test/scala/za/co/absa/spline/DeltaDSV2Spec.scala

```diff
@@ -345,7 +345,7 @@ class DeltaDSV2Spec extends AsyncFlatSpec
             plan.operations.write.outputSource shouldBe s"file:$warehouseDir/testdb.db/foo"
 
             val mergeOp = plan.operations.write.precedingDataOp
-            mergeOp.params.get("condition").asInstanceOf[String] should include("ID")
+            mergeOp.params.get("condition").asInstanceOf[Option[String]].get should include("ID")
```

Comment:
I guess it's better to use ScalaTest's `OptionalValues` implicits for matching options.
https://www.scalatest.org/user_guide/using_OptionValues